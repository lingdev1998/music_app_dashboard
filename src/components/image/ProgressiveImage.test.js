import React from 'react';
import { shallow } from 'enzyme';
import ProgressiveImage from './progressiveImage';

describe('<ProgressiveImage />', () => {
    const ImageWrapper = shallow(<ProgressiveImage />);
    it('matches snapshot', () => {
        expect(ImageWrapper).toMatchSnapshot()
    })
})