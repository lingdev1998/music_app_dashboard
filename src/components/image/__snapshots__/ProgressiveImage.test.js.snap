// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressiveImage /> matches snapshot 1`] = `ShallowWrapper {}`;
